rele/middleware.py

```diff
@@ -6,7 +6,7 @@
 default_middleware = ["rele.contrib.LoggingMiddleware"]
 
 
-def register_middleware(config):
+def register_middleware(config, **kwargs):
```

Comment:
I don't see how this is going to solve the issue. `register_middleware` is called from `config.setup`, so it won't get any parameters anyway. We would need an extra mechanism for this.

Reply:
OK, let's do it like this